Describe wrapperAttr and e.cancel in onShowing for overlays (#2604)
DevExpress/devextreme-documentation: api-reference/10 UI Components/dxOverlay/1 Configuration/onHiding.md

Information about the event.

##### field(e.cancel): Boolean

> Set this field to **true** if you want the {WidgetName} to be displayed.

Reviewer: ```suggestion
Set this field to **true** if you want the {WidgetName} to stay on the screen.
```

> Set this field to **true** if you want the {WidgetName} to stay on the screen.

Reviewer 2: ```suggestion
Set this field to **true** if you want the {WidgetName} to remain visible.
```
